File: AP Computer Science A 2018-19/Object-Oriented Programming/OOP Class Extension Demo/Person.java
public class Person {

	private String name;
	private String alliance;

	public Person() {
		this.name = "no name";
		this.alliance = "no alliance";
	}

	public Person(String name, String alliance) {
		this.name = name;
		this.alliance = alliance;
	}

	public String getName() { return name; }

	public String toString() { return "The name of the person is " + name + "."; }

}

File: VR and Game Development 2020-21/NotesDay2/Person.cs
using System;
namespace NotesDay2
{
    class Person
    {
        private string firstName;
        private string eyeColor;
        private int daysAlive;

        public string LastName { get; set; }

        public float YearsAlive
        {
            get
            {
                return (float)(daysAlive / 365.0);
            }

            set
            {
                daysAlive = (int)(value * 365);
            }
        }

        public Person(string firstName, int daysAlive)
        {
            this.firstName = firstName;
            this.daysAlive = daysAlive;
        }

        // Constructor Chaining
        public Person(string firstName, string eyeColor, int daysAlive)
            :this(firstName, daysAlive)
        {
            this.eyeColor = eyeColor;
        }

        public override string ToString()
        {
            return firstName + " " + LastName + " has " + eyeColor + " eyes and has been alive for " + YearsAlive + " years.";
        }
    }

}
